frontend/public/components/app.jsx

```diff
@@ -214,8 +214,8 @@ graphQLReady.onReady(() => {
   // Global timer to ensure all <Timestamp> components update in sync
   setInterval(() => store.dispatch(UIActions.updateTimestamps(Date.now())), 10000);
 
-  // Fetch swagger on load if it's stale.
-  fetchSwagger();
+  // Poll swagger definitions to ensure they stay up to date
+  setInterval(fetchSwagger, 30000);
```

Comment:
Looks like the commit message doesn't match the timeout in the code.

We might scale this back to 1 minute so we're not hitting the server quite as often. Another try would be to wait 15s or 30s after API services change.

Reply:
Okay, I updated to do both. Definitions get updated by a poll every minute and every time API discovery occurs.

Reply:
If 10s is enough of a delay to have the doc update, we probably want to increase the poll timeout since it's not so important.

Reply:
Yeah I tested 10s with a smaller CRD, but I might want to try creating one that has a heftier OpenAPI definition and see if it still works.

Reply:
Okay, tested wit Portworx which has a pretty large OpenAPI definition, and that took 15 to 25 seconds to update. I should probably set this to something higher than 10, but not so high that it won't have a chance to update before users go to create a CRD instance. What sounds reasonable to you @spadgett?

Reply:
Actually tested a bit more, and even with Portworx, a 10 second delay seems to work consistently.